lib/compat/Makefile.am

```diff
@@ -17,6 +17,7 @@ compat_sources			= 	\
 	lib/compat/inet_aton.c		\
 	lib/compat/memrchr.c		\
 	lib/compat/pio.c		\
+	lib/compat/glib.c		\
```

Comment:
You should add `glib.c` also to the `lib/compat/CMakeLists.txt`.

Reply:
Fixed